docs/src/increase-observability/metrics.md

```diff
@@ -0,0 +1,252 @@
+
+For this example, each host has been granted 4&nbsp;GB of memory.
+The upper limit in the graph is 3.62&nbsp;GB because 0.38&nbsp;GB of memory are in use by the operating system.
+
+![A closeup of the RAM metrics](/images/metrics/ram.png "0.5")
+
+#### Disk
+
+Disk represents the plan space for all services and mounts,
+which for Dedicated production environments is [50&nbsp;GB](../dedicated/architecture/_index.md).
+
+This example has the following persistent disk space configured:
+
+* App: 1024&nbsp;MB for the mount `exports`
+* Elasticsearch: 1024&nbsp;MB
+* MySQL: 2048&nbsp;MB
+* Redis: no persistent disk
+
+![All of the metrics for disks](/images/metrics/disk.png)
+
+For each host, you first see metrics for the root (`/`) folder.
+Utilizing this folder doesn't count against the 50&nbsp;GB set aside for your plan:
+it's infrastructure disk, including the `/tmp` directory.
+It's included in the metrics so you can keep track of build artifacts you write to `/tmp`
+and make sure that they don't get too large and fill up.
+
+Next to this space comes the other defined directories: for the MySQL service and the `exports` mount.
+
+![A closeup of the metrics for the MySQL service disk](/images/metrics/disk-single.png "0.4")
+
+### Grid environment example
+
+Grid environment metrics show resource usage for each app, service, and worker container.
+
+This reference project has configured a single application and two services: Redis and MariaDB.
+The plan size for this project is [X-Large](https://platform.sh/pricing/).
+The appropriate resources have been [allocated automatically](../configuration/app/app-reference.md#sizes) for each container
+based on the number and type of containers for this plan size.
+The graphs show the current average usage in relation to the allocated resources.
+
+#### App container
+
+Metrics graphs for the app container show CPU, RAM, and disk allocation and usage.
+The persistent disk has been configured in the [app configuration](../configuration/app/app-reference.md#top-level-properties)
+at 144.12&nbsp;GB, while the temporary disk is 3.99&nbsp;GB by default.
+
+![All of the metrics for the app container](/images/metrics/app-container.png)
+
+#### Service containers
+
+##### MySQL
+
+Metrics graphs for the MySQL service container show CPU and disk allocation and usage.
+The persistent disk has been configured in the [services configuration](../configuration/services/_index.md) as 4.74&nbsp;GB (5000MB),
+while the temporary disk is 3.99&nbsp;GB by default.
+
+![All of the metrics for the MySQL container](/images/metrics/mysql-container.png)
+
+##### Redis
+
+Metrics graphs for the Redis service container show CPU and disk allocation and usage.
+No persistent disk has been configured in the [services configuration](../configuration/services/_index.md),
+while the temporary disk is 3.99&nbsp;GB by default.
+
+![All of the metrics for the Redis container](/images/metrics/redis-container.png)
```

Comment:
##### Solr
Metrics graphs for the Solr service container show CPU and disk allocation and usage.
The persistent disk has been configured in the [services configuration](../configuration/services/_index.md) at 0.48 GB, while the temporary disk is 3.99&nbsp;GB by default.


<img width="1384" alt="Screenshot 2022-03-16 at 10 49 57" src="https://user-images.githubusercontent.com/45591272/158563227-e027eae1-95fd-446a-9aa0-96365349dd3a.png">